virtual_workstation_setup.sh

```diff
@@ -100,6 +102,29 @@ setup_cr() {
     popd
 }
 
+setup_jira_auth() {
+    # Get the user's JIRA username
+    read -p "JIRA username (from https://jira.mongodb.org/secure/ViewProfile.jspa): " jira_username
+    echo "export JIRA_USERNAME=$jira_username" >> ~/.bash_profile
+    echo "Wrote username \"$JIRA_USERNAME\" to ~/.bash_profile"
+    export JIRA_USERNAME=$jira_username
+
+    # Set up the Jira OAuth Token Generator repo
+    pushd $HOME/mongodb-mongo-master
+        git clone git@github.com:10gen/iteng-jira-oauth.git
+        mkdir venv
+        /opt/mongodbtoolchain/v3/bin/python3 -m venv venv
+
+        # Get credentials and store them in the system keyring
+        source venv/bin/activate
+            python -m pip install --upgrade pip
+            python -m pip install -r iteng-jira-oauth/requirements.txt
```

Comment:
Consider putting the venv in `iteng-jira-oauth` so it's clearer what it's used for..

Reply:
done